test/e2e/e2e_test.go

```diff
@@ -131,6 +132,66 @@ var _ = ginkgo.Describe("JobSet", func() {
 		})
 	})
 
+	ginkgo.When("job is suspended and resumed", func() {
+
```

Comment:
Maybe you could add a comment here about this e2e test being more kueue integration.

Reply:
Done